### PR TITLE
v1.7.2

### PR DESCRIPTION
**Pull request by bartlomieju**
<s>Blocked by https://github.com/denoland/deno/pull/9410</s>

### CODE REVIEW

File: Releases.md [hunk 1]
```diff
@@ -6,6 +6,23 @@ https://github.com/denoland/deno/releases
...
+- feat(lsp): add TS quick fix code actions (#9396)
+- fix: improve http client builder error message (#9380)
+- fix(cli): fix handling of non-normalized specifier (#9357)
+- fix(cli/coverage): display mapped instrumentation line counts (#9310)
+- fix(cli/lsp): fix using jsx/tsx when not emitting via tsc (#9407)
+- fix(repl): prevent symbol completion panic (#9400)
+- refactor: rewrite Blob implementation (#9309)
+- refactor: rewrite File implementation (#9334)
+
+Changes in std version 0.86.0:
```
Comment (ry) [inline Releases.md:21]: Maybe we should move the std release notes to the std repo?

File: Releases.md [hunk 2]
```diff
@@ -6,6 +6,23 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at:
 https://github.com/denoland/deno_install
 
+### 1.7.2 / 2021.02.05
+
+- feat(lsp): add references code lens (#9316)
+- feat(lsp): add TS quick fix code actions (#9396)
```
Comment (ry) [inline Releases.md:12]: feat(lsp, unstable) ? (just to deflect inevitable complaints we're not sticking to semver?)

File: Releases.md (hunk 1 above)
Comment (bartlomieju) [reply Releases.md:21]: I'm open to that, but we should at least link it here

File: Releases.md (hunk 2 above)
Comment (bartlomieju) [reply Releases.md:12]: Sure, I'll rework it

File: Releases.md (hunk 1 above)
Comment (ry) [reply Releases.md:21]: Ok

**Review by ry — APPROVED** [review]

**Pull request merged**